swig/README.md

```diff
@@ -169,22 +169,37 @@ iter = 29, loss = 8.881784E-16
 ## Usage
 
 The Scala version of DyNet is intended to work mostly like the
-C++. However, there are a few exceptions to watch out for, which are
+C++. However, there are a few things to watch out for, which are
 documented below.
 
 ### Imports
 
-Import dynet using:
+All of the DyNet classes and structs are in the `edu.cmu.dynet` package.
+DyNet also contains a large number of bare functions, they end up as
+static methods on the `dynet_swig` class. Many of them have common names
+(e.g. `sum`), so you probably don't want to pollute your namespace by 
+importing them all. Our convention is to rename that class `dn`.
+Finally, the additional Scala helpers are contained in the `DyNetScalaHelpers`
+object.
+
+So a typical usage looks like:
 
 ```scala
-// Dynet classes and structs, e.g., Parameter
-import edu.cmu.dynet._
-// Dynet neural network operations, e.g., affine_transform
-import edu.cmu.dynet.dynet_swig._
-// Scala helpers, e.g., implicit type conversions
-import edu.cmu.dynet.DynetScalaHelpers._
+import edu.cmu.dynet.{dynet_swig => dn, _}
```

Comment:
IntelliJ did that for me automatically, because I had so many imports.

e.g. Twitter's "effective scala" says

"Use wildcards when more than six names are imported"

although with a caveat

http://twitter.github.io/effectivescala/#Formatting-Imports

should I make them explicit even if that means a huge list of imports?